### PR TITLE
Update recommended IDE

### PR DESCRIPTION
**Pull request by kaixin-hc**
**What is the purpose of this pull request?**

- [X] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Added VS code to recommendations, because all the rest of our docs have guides for webstorm OR vs code, and I think most of the developer team now uses vs code only.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**

Update documentation for recommended IDE

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [X] No unrelated changes 

---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:
- [ ] Major (when you make incompatible API changes)
- [ ] Minor (when you add functionality in a backward compatible manner)
- [X] Patch (when you make backward compatible bug fixes)

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).

### CODE REVIEW

File: docs/devGuide/development/settingUp.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This page explains how to set up your development environment to start contribut
 
 <box type="tip" seamless>
 
-We recommend the **WebStorm IDE** for working with MarkBind code.
+We recommend the **WebStorm IDE** or **VS Code** for working with MarkBind code.
```
Comment (yucheng11122017) [inline docs/devGuide/development/settingUp.md:37]: Small nit: should we include links to both IDEs so its easier for users?

**Review by yucheng11122017 — APPROVED** [review]
LGTM except for a small nit!

File: docs/devGuide/development/settingUp.md (hunk 1 above)
Comment (kaixin-hc) [reply docs/devGuide/development/settingUp.md:37]: It's quite silly but when I thought about it I wasn't sure if I liked the look of it being being bolded and a link 😂 I was also thinking about if the user would expect it to link to something within the MarkBind website and be taken by surprise by a link that goes out

Comment (yucheng11122017) [reply docs/devGuide/development/settingUp.md:37]: Fair enough

**Pull request merged**